packages/samples/common-types/openapi/v4/types.json

```diff
@@ -506,7 +471,7 @@
           "readOnly": true
         },
         "systemData": {
-          "$ref": "#/definitions/SystemData",
+          "$ref": "#/definitions/systemData",
```

Comment:
verify if this change doesn't impact SDK team, Wei Hu

Reply:
in this case typespec is changed, is this ok? or do I need to create a special case in where this is changed on post processing and in tsp we keep upper case

Reply:
If we are generating SDK from swagger, this will cause issue for SDK generation for all languages.
Because you changed the definition id here, and the other services referencing to this(with upper case) will not be able to find it. Modular 4 will not be able to compile the swagger, the SDK generation will fail in the first step.

Reply:
yes, we are keeping this to lower case (same as original swagger)  https://github.com/Azure/azure-rest-api-specs/blob/e256596e6701c96989581b97909a773f2520074f/specification/common-types/resource-management/v4/types.json#L381

my question is related to TypeSpec side, does it matter the casing of TypeSpec?

Reply:
For TypeSpec input, I think the first letter should be fine, all language emitters should have taken care of them already. 
We should not change the casing within the word as it would cause word separation changes.
I can double confirm with the team.

Reply:
Confirmed with all language owners, the SDK generation will not be affected.
But we are wondering why we are changing the model names directly, can we keep the names upper case and use `@encdoeName` to change the generated swagger to lower case?